pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -426,7 +430,11 @@ func (r *MigPlan) UpdateRegistryDC(storage *MigStorage, deploymentconfig *appsv1
 							},
 							kapi.EnvVar{
 								Name:  "REGISTRY_STORAGE_S3_REGION",
-								Value: storage.Spec.BackupStorageConfig.AwsRegion,
+								Value: region,
+							},
+							kapi.EnvVar{
+								Name:  "REGISTRY_STORAGE_S3_REGIONENDPOINT",
+								Value: storage.Spec.BackupStorageConfig.AwsS3URL,
```

Comment:
Have you confirmed that this still works when REGIONENDPOINT is empty? I'm wondering whether we should only set this var when we have a value for it.

Reply:
Yes, when `AwsS3URL` is an empty string the value is not set inside of the DC. I have tested using minio (specifying an endpoint) and using AWS S3 (no endpoint) and saw no trouble

Reply:
Example if the endpoint URL is not specified

```
      spec:                    
        containers:      
        - env:         
          - name: REGISTRY_STORAGE                    
            value: s3                   
          - name: REGISTRY_STORAGE_S3_ACCESSKEY
            valueFrom:                  
              secretKeyRef:                    
                key: access_key
                name: registry-fgdgs-fkf2p
          - name: REGISTRY_STORAGE_S3_BUCKET
            value: velero-5f3e46ba0dc70309b91132c3db936347
          - name: REGISTRY_STORAGE_S3_REGION
            value: us-east-2
          - name: REGISTRY_STORAGE_S3_REGIONENDPOINT
          - name: REGISTRY_STORAGE_S3_ROOTDIRECTORY
            value: /fgdgs-registry-78e01032-bec9-11e9-842d-0a60c6315958
          - name: REGISTRY_STORAGE_S3_SECRETKEY
```